esmvalcore/_recipe.py

```diff
@@ -302,16 +303,10 @@ def _get_default_settings(variable, config_user, derive=False):
         'mip': variable['mip'],
         'short_name': variable['short_name'],
         'frequency': variable['frequency'],
-        'check_level': config_user['check_level']
+        'check_level': config_user.get('check_level', CheckLevels.DEFAULT)
     }
     # Configure final CMOR data check
-    settings['cmor_check_data'] = {
-        'cmor_table': variable['project'],
-        'mip': variable['mip'],
-        'short_name': variable['short_name'],
-        'frequency': variable['frequency'],
-        'check_level': config_user['check_level']
-    }
+    settings['cmor_check_data'] = settings['cmor_check_metadata']
```

Comment:
```suggestion
    settings['cmor_check_data'] = dict(settings['cmor_check_metadata'])
```
I would feel safer if these were not the same dict, though it will probably work fine as is

Reply:
Yeah good point, I was wondering about this too!